wdl-lint/src/rules/description_missing.rs

```diff
@@ -0,0 +1,81 @@
+//! A lint rule for to ensure description is included in meta sections.
+
+use wdl_ast::v1::MetadataSection;
+use wdl_ast::AstNode;
+use wdl_ast::AstToken;
+use wdl_ast::Diagnostic;
+use wdl_ast::Diagnostics;
+use wdl_ast::Span;
+use wdl_ast::ToSpan;
+use wdl_ast::VisitReason;
+use wdl_ast::Visitor;
+
+use crate::Rule;
+use crate::Tag;
+use crate::TagSet;
+
+/// The identifier for the description missing rule.
+const ID: &str = "DescriptionMissing";
+
+/// Creates a description missing diagnostic.
+fn description_missing(span: Span) -> Diagnostic {
+    Diagnostic::note("description key missing in meta section")
+        .with_rule(ID)
+        .with_highlight(span)
+        .with_fix("add a description to the meta section")
```

Comment:
```suggestion
        .with_fix("add a `description` key to the meta section")
```

I might consider splitting this into two different diagnostics: one for tasks and one for workflows:

- "add a `description` key to the meta section describing the task"
- "add a `description` key to the meta section describing the workflow"